packages/widgets/src/lib/components/InferenceWidget/shared/helpers.ts

```diff
@@ -124,8 +120,8 @@ export async function callInferenceApi<T>(
 		headers.set("X-Load-Model", "0");
 	}
 
-	const reqBody: File | string =
-		"file" in requestBody && requestBody["file"] instanceof File ? requestBody.file : JSON.stringify(requestBody);
+	const reqBody: Blob | string =
```

Comment:
```suggestion
	// `File` is a subtype of `Blob`: therefore, checking for instanceof `Blob` also checks for instanceof `File`
	const reqBody: Blob | string =
```

I'd just add this comment